public/editor/scripts/editor/js/fc/project-rename.js

```diff
@@ -132,15 +133,20 @@ define(function(require) {
         }
         editingComplete(context);
         analytics.event("ProjectRenamed");
+
+        var publisher = new Publisher();
+        publisher.init(bramble);
```

Comment:
This is failing on Travis:

```
   public/editor/scripts/editor/js/fc/project-rename.js

    138 |        publisher.init(bramble);
                                ^ 'bramble' is not defined.
```

The `bramble` object will have to get passed in here somehow.